lib/tasks/send_supervisor_digest.rake

```diff
@@ -0,0 +1,6 @@
+desc "Send an email to supervisors each week to share an overview of their volunteers' activities"
+task send_supervisor_digest: :environment do
```

Comment:
We'll need a cron for this or similar- I assume heroku has something for this?

Reply:
I wrote a mailer for reminders when court reports are due.  I believe I was told to create a task that would be run by heroku.  I never followed up about that to see if that was scheduled.  I would assume heroku has something for that, but I'm not super familiar.  I can look it up and try to set that up once this gets merged so we can set it up for this one and the court report reminder.